docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -125,6 +197,10 @@ components:
   # Resusable resource schemas
   # ------------------------------
   schemas:
+    Annotation:
+      $ref: './schemas/annotation-schema.json'
+    AnnotationCreate:
+      $ref: './schemas/annotation-schema.json'
```

Comment:
These two types definitely shouldn't be the same. Is that what you meant about inaccuracies in the annotation response? For example, the docs for the annotation response (eg. `GET /annotations/{id}`) don't mention the ID or timestamp fields (!)

Reply:
This is indeed what I mean about the `annotation` schema being completely wrong in our current docs. These new docs share the same underlying resource schema (for now, at least). Yeah, it needs fixing, but it's orthogonal.